paths/paths.go

```diff
@@ -26,6 +26,11 @@ func DefaultTailscaledSocket() string {
 	if runtime.GOOS == "darwin" {
 		return "/var/run/tailscaled.socket"
 	}
+	// TODO(crawshaw): does this path change with DSM7?
+	const synologySock = "/volume1/@appstore/Tailscale/var/tailscaled.sock" // SYNOPKG_PKGDEST in scripts/installer
+	if fi, err := os.Stat(filepath.Dir(synologySock)); err == nil && fi.IsDir() {
```

Comment:
can you at least do this conditionally on runtime.GOOS == "linux" so we don't do all these stats on other operating systems?